frontend/src/components/Bill/LocForm.jsx

```diff
@@ -131,10 +127,10 @@ const LocForm = ({ sponsorNames }) => {
           {Object.entries(representatives).map(([name, rep]) => (
             <div key={rep.name} className="representative">
               <h3>
-                {rep.title} {rep.name}:{' '}
-                {sponsorshipStatus[rep.name]
+                {rep.title} {rep.name}
+                {/* {sponsorshipStatus[rep.name]
                   ? 'Already a sponsor'
-                  : 'NOT A SPONSOR'}
+                  : 'NOT A SPONSOR'} */}
```

Comment:
Remove comment or add context to why it's here.

Reply:
I'll add context for which representatives the bill page shows and whether or not they are already a co-Sponsor